terraform/exoscale/modules/kubernetes-cluster/main.tf

```diff
@@ -100,7 +100,7 @@ resource "exoscale_compute" "nfs" {
   display_name    = "${var.prefix}-nfs"
   template_id     = data.exoscale_compute_template.ubuntu.id
   size            = var.nfs_size
-  disk_size       = 200
+  disk_size       = var.nfs_disk_size == 0 ? 200 : var.nfs_disk_size
```

Comment:
Same here

Reply:
Nvm, this is needed to be backwards compatible since `nfs_disk_size` is going to be set to the zero value when tfvars.json is rendered (e.g. when you add a new machine) if it wasn't set before.

We don't really have a way of controlling default values except for when initializing the configuration. Might be a good idea to find a way to control that. This is fine for now!